src/uwtools/api/config.py

```diff
@@ -0,0 +1,77 @@
+# pylint: disable=unused-import
+
+from typing import Optional
+
+from uwtools.config.atparse_to_jinja2 import convert as _convert_atparse_to_jinja2
+from uwtools.config.tools import compare_configs as _compare
+from uwtools.config.tools import realize_config as _realize
+from uwtools.config.validator import validate_yaml as _validate
+from uwtools.types import DefinitePath, OptionalPath
+from uwtools.utils.file import FORMAT
+
+
+def compare(
+    config_a_path: DefinitePath,
+    config_a_format: str,
+    config_b_path: DefinitePath,
+    config_b_format: str,
+) -> bool:
+    """
+    ???
+    """
```

Comment:
Deferring docstring writing until [UW-443](https://jira.epic.oarcloud.noaa.gov/browse/UW-443) is worked, to ensure that the docstrings are written together to ensure a consistent style. These will be the docstrings that we will hopefully be able to use Sphinx to auto-generate API documentation from.